File: solutions/0009-palindrome-number/solution.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        temp = x
        res = 0
        if x < 0: return False
        while temp!=0:
            rem = temp %10
            res = 10*res + rem
            temp = temp // 10
        return res == x

File: solutions/0009-palindrome-number/solution.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if(x<0) return false;
        int n = x;
        long long res = 0;
        while(x > 0){
            res = 10*res + (x%10);
            x/=10;
        }
        return res == n;
    }
};
